wiki/Tournaments/VTSS/1/en.md

```diff
@@ -0,0 +1,1150 @@
+12. Bans may not double ban a mod (if player 1 already banned HD1, they can't ban HD2 with their second ban, however player 2 still can ban HD2). NM is the exception and may be double banned by the same player.
+13. Once a player reached the required amount of wins, the match ends and the player who reached that amount is declared the winner.
+14. The procedures for protects, bans, and picks are:
+
+| Round of 128 to Round of 32 | Round of 16 to Grand Finals |
+| :-- | :-- |
+| 1. Player 1 protects | 1. Player 1 protects |
+| 2. Player 2 protects | 2. Player 2 protects |
+| 3. Player 2 bans | 3. Player 2 bans |
+| 4. Player 1 bans | 4. Player 1 bans |
+| 5. Player 1 picks | 5. Player 2 bans |
+| 6. Player 2 picks (alternate until a winner is declared or tiebreaker is reached) | 6. Player 1 bans |
+|  | 7. Player 1 picks |
+|  | 8. Player 2 picks (alternate until a winner is declared or tiebreaker is reached) |
+
+15. Before each map, each player has **two minutes** before the match is played unless both parties with to play the map after agreed time.
+16. All maps will be played with NoFail mod and Score V2 alongside the map's respective mods.
+17. Hidden, HardRock, and DoubleTime maps will have their respective mods chosen by the referee.
+18. When a FreeMod map is played, each player must at least pick one mod from the following: Easy, Hidden, HardRock, or Flashlight.
+    - Easy receives a `1.75x` modifier on top of the `0.5x` score modifier, making it `0.875x` total score.
+19. In case of a tie in points (example: 4-4 in BO9) the tiebreaker is selected.
+    - Players may pick any amount of the following mods: Easy, Hidden, HardRock, or Flashlight. However it's not required and they may play their tiebreaker with NoMod.
+
+### Mappool
+
+| Stage | Highest Star Rating | Maps |
+| :-- | :-: | :-- |
+| Qualifiers | 5.3 ★ | 4 NoMod, 2 Hidden, 2 HardRock, 2 DoubleTime |
+| Round of 128 | 4.75 ★ | 4 NoMod, 2 Hidden, 2 HardRock, 2 DoubleTime, 2 FreeMod, 1 Tiebreaker |
+| Round of 64 | 4.95 ★ | 4 NoMod, 2 Hidden, 2 HardRock, 2 DoubleTime, 2 FreeMod, 1 Tiebreaker |
+| Round of 32 | 5.15 ★ | 5 NoMod, 2 Hidden, 2 HardRock, 3 DoubleTime, 2 FreeMod, 1 Tiebreaker |
+| Round of 16 | 5.35 ★ | 5 NoMod, 2 Hidden, 2 HardRock, 3 DoubleTime, 2 FreeMod, 1 Tiebreaker |
+| Quarterfinals | 5.55 ★ | 5 NoMod, 3 Hidden, 3 HardRock, 3 DoubleTime, 2 FreeMod, 1 Tiebreaker |
+| Semifinals | 5.65 ★ | 5 NoMod, 3 Hidden, 3 HardRock, 3 DoubleTime, 2 FreeMod, 1 Tiebreaker |
+| Finals | 5.75 ★ | 6 NoMod, 3 Hidden, 3 HardRock, 3 DoubleTime, 3 FreeMod, 1 Tiebreaker |
+| Grand Finals | 5.75 ★ | 6 NoMod, 3 Hidden, 3 HardRock, 3 DoubleTime, 3 FreeMod, 1 Tiebreaker |
+
+*Note: Highest star rating in the mappool will always be NM1.*
+
+### Match incidents during play
+
+| Incident | Punishment |
+| :-- | :-- |
+| Player doesn't choose a map to protect/ban within 2 minutes | The player loses the right to protect or ban the map (depending on the situation) and move onto the next step. |
```

Comment:
"... and move onto the next step" (the right which they lose as well), or "and move**S** onto the next step" (meaning their ban/protect turn is skipped)?